src/check_cve/check.py

```diff
@@ -65,23 +70,51 @@ def is_vulnerable(host):
 def main():
     """Parse arguments, handle logging, and call vulnerability check."""
     args = docopt.docopt(__doc__, version=__version__)
-    # Set up logging
-    log_level = args["--log-level"]
+
+    # Validate and convert arguments as needed
+    schema = Schema(
+        {
+            "--log-level": And(
+                str,
+                Use(str.lower),
+                lambda n: n in ("debug", "info", "warning", "error", "critical"),
+                error="Possible values for --log-level are "
+                + "debug, info, warning, error, and critical.",
+            ),
+            "--retries": And(
+                Use(int),
+                lambda n: 0 <= n,
+                error="--retries must be an integer greater than 0.",
+            ),
+            "--timeout": And(
+                Use(int),
+                lambda n: 1 <= n,
+                error="--timeout must be a an integer greater than 1.",
```

Comment:
Your error string says "greater than 1" but your lambda checks if it is greater than _or equal to_ one.

Reply:
Good catch.  Things got messed up when I converted my language.  I used to say "positive integers", but then realized that didn't capture zero.  Anyway, updated lambda logic, English error message, and the pytests for them in 8380206. 

Crikey... I just messed it up again.  brb.

Reply:
Ok.  All good now: bbff80e
But you should double-check my logic.  I must be hungry.